modules/claims_api/app/swagger/claims_api/claims_v1_swagger.rb

```diff
@@ -54,41 +54,10 @@ class ClaimsV1Swagger
         end
       end
 
-      key :servers, [
-        {
-          "url": 'https://dev-api.va.gov/services/claims/{version}',
-          "description": 'VA.gov API development environment',
-          "variables": {
-            "version": {
-              "default": 'v0'
-            }
-          }
-        },
-        {
-          "url": 'https://staging-api.va.gov/services/claims/{version}',
-          "description": 'VA.gov API staging environment',
-          "variables": {
-            "version": {
-              "default": 'v0'
-            }
-          }
-        },
-        {
-          "url": 'https://api.va.gov/services/claims/{version}',
-          "description": 'VA.gov API production environment',
-          "variables": {
-            "version": {
-              "default": 'v0'
-            }
-          }
-        }
-      ]
-
       security_definition :bearer_token do
-        key :type, :http
-        key :name, :bearer_token
-        key :scheme, :bearer
-        key :bearerFormat, :JWT
+        key :type, :apiKey
```

Comment:
You've probably explained this before but why are we listing an apiKey in a Claims V1 bearer_token instead of an actual bearer token?